src/test/regress/expected/shard_rebalancer.out

```diff
@@ -1351,25 +1355,41 @@ WHERE table_schema = 'public'
 ) a ORDER BY table_name;
  table_schema | table_name  | row_estimate | total_bytes
 ---------------------------------------------------------------------
- public       | tab2_123051 |        10000 |      393216
- public       | tab2_123052 |        10000 |      393216
- public       | tab2_123053 |        60000 |     2203648
- public       | tab_123034  |        10000 |      393216
- public       | tab_123035  |        10000 |      368640
- public       | tab_123036  |        10000 |      393216
+ public       | tab2_123051 |        10000 |      368640
+ public       | tab2_123052 |        10000 |      368640
+ public       | tab2_123053 |        60000 |     2179072
+ public       | tab_123041  |        10000 |      368640
+ public       | tab_123042  |        10000 |      368640
+ public       | tab_123043  |        10000 |      368640
 (6 rows)
 
 \c - - - :master_port
 SELECT * FROM get_rebalance_table_shards_plan('tab', rebalance_strategy := 'by_disk_size');
+NOTICE:  Ignoring move of shard xxxxx from localhost:xxxxx to localhost:xxxxx, because the move only brings a small improvement relative to the shard its size
+NOTICE:  Ignored 1 moves, all of which are shown in notices above
+HINT:  If you do want these moves to happen, try changing improvement_threshold to a lower value than what it is now (0.5).
```

Comment:
how hard would it be to include the actual utilization threshold value of the ignored move. eg. showing the user the move was ignored due to the utilization improvement was 0.4, together with the ability to forcefully show all moves it can guide is into where the utilization could be placed to allow certain moves to be considered.

I guess we already have the value, given we are testing against the configured vaule.
The extra information probably can be added in a detail field of the notice?

Now it will be hard to guess the threshold to use when tweaking.

Reply:
Would be easy to add, because we indeed have the value already.